api/src/main/java/org/openmrs/module/owa/AppOpenmrs.java

```diff
@@ -38,16 +38,27 @@
 	/**
 	 * Determines if a de-serialized file is compatible with this class.
 	 */
-	private static final long serialVersionUID = -8854371580010728182L;
```

Comment:
What is the reason behind changing this?

Reply:
I added a new field `AppRequirements` that basically changes the compositions of this class to a different unique state. I had to generate a new serial value for this class that currently uniquely identifies it basing on it's real composition as was initially done by the original authors previously, hence preserving the original intention of providing it as stated in the comment above this field. I used JDK tool, `Serialver` to generate it.